core/connection/manager.go

```diff
@@ -127,13 +128,14 @@ func (manager *connectionManager) startConnection(consumerID, providerID identit
 	}
 	cancel = append(cancel, func() { dialog.Close() })
 
-	vpnSession, err := session.RequestSessionCreate(dialog, proposal.ID)
+	vpnSession := session.Session{Config: openvpn_node.VPNConfig{}}
+	err = session.RequestSessionCreate(dialog, proposal.ID, &vpnSession)
 	if err != nil {
 		return err
 	}
 
 	stateChannel := make(chan openvpn.State, 10)
-	openvpnClient, err := manager.startOpenvpnClient(*vpnSession, consumerID, providerID, stateChannel, options)
+	openvpnClient, err := manager.startOpenvpnClient(vpnSession, consumerID, providerID, stateChannel, options)
```

Comment:
Description of the `session.Session` says:
> // Session structure holds all required information about current session between service consumer and provider

Maybe `consumerID` and `providerID` can be the part of the `vpnSession` to avoid passing duplicate args?

Reply:
@vkuznecovas is working in another PR to have `ConnectionParams` structure